pkg/gor/debian/rules

```diff
@@ -2,6 +2,7 @@
 
 export GOPATH=$(CURDIR)
 export GOBIN=$(CURDIR)/debian/gor/usr/bin
+export VERSION="0.10.1"
```

Comment:
Instead of duplicating this here, it should be possible to get this from the package versioning.  I think debhelper already sets up `DEB_UPSTREAM_VERSION` for you.